pkg/client/client.go

```diff
@@ -177,9 +177,10 @@ func (a AppUpdateOptions) ToRun() AppRunOptions {
 }
 
 type ImageProgress struct {
-	Total    int64  `json:"total,omitempty"`
-	Complete int64  `json:"complete,omitempty"`
-	Error    string `json:"error,omitempty"`
+	Total       int64  `json:"total,omitempty"`
+	Complete    int64  `json:"complete,omitempty"`
+	Error       string `json:"error,omitempty"`
+	CurrentTask string `json:"currentTask,omitempty"`
```

Comment:
I added this field so that we can tell the user what is currently happening. This is very useful in particular for `--all-tags`.

Reply:
If I do `acorn copy --all-tags <source> <dest>` and then hit cancel out (say through `ctrl`+`c`), does the copy continue happening in the background?

Reply:
Just tested this to see. The copy terminates properly and does not continue in the background